master/docs/guides/svn.rst

```diff
@@ -0,0 +1,92 @@
+=================================
+Subversion Version Control System
```

Comment:
That does seem like a lot of guides, it's true.  Maybe I just need to adjust the expectation I have from the word "guide" -- these are each focused on a particular usage topic that might involve content from all over the documentation (particularly for version-control systems, which will reference both change sources and source steps).